### PR TITLE
You May Now Kiss The Girlboss

### PR DESCRIPTION
**Pull request by Sun-Soaked**
## About The Pull Request
Ports the *Kiss emote. When used inhand it blows a kiss at the target, giving a mood boost if it strikes.
Giving the emote "item" to someone allows you to kiss them directly.
Restructures Give code, it's (allegedly) more modular this way.

Adds the Kiss of Death adminspawn.

This one was a bit of a mess to port, make sure to let me know all the ways it breaks.

Ported from [(here)](https://github.com/tgstation/tgstation/pull/56698) and [(here)](https://github.com/tgstation/tgstation/pull/61590).

## Why It's Good For The Game
When the moon hits your eye like a big pizza pie,

## Changelog

:cl: Ryll-Ryll, Memed Hams
add: *kiss emote, can be "blown" at people as a projectile, or given as a peck on the cheek.
add: The Kiss of Death, a legendary lipstick rumoured to provide heart-stopping beauty.
code: emote items moved to the emotes.dm file
code: ports tg's rework of the give system
/:cl:

### CODE REVIEW

**Comment by Fikou** [comment]
credit the original creator in the changelog please

File: code/_onclick/hud/alert.dm [hunk 1]
```diff
@@ -291,57 +291,98 @@ or shoot a gun to move around via Newton's 3rd Law of Motion."
...
-	desc = "[giver] is offering [receiving]. Click this alert to take it."
+	name = "[offerer] is offering [receiving]"
+	desc = "[offerer] is offering [receiving]. Click this alert to take it."
 	icon_state = "template"
 	cut_overlays()
 	add_overlay(receiving)
 	src.receiving = receiving
-	src.giver = giver
+	src.offerer = offerer
 	RegisterSignal(taker, COMSIG_MOVABLE_MOVED, .proc/removeAlert)
```
Comment (ZephyrTFA) [inline code/_onclick/hud/alert.dm]: Get rid of this line you immediately override it in the very next line

**Review by ZephyrTFA — CHANGES_REQUESTED** [review]

File: code/_onclick/hud/alert.dm (hunk 1 above)
Comment (Sun-Soaked) [reply code/_onclick/hud/alert.dm]: aye captain

**Comment by ZephyrTFA** [comment]
You removed the wrong line

**Comment by Chirynnn** [comment]
one step closer to erp

File: code/modules/mob/living/emote.dm [hunk 2]
```diff
@@ -197,9 +197,24 @@
...
+		return
+	var/kiss_type = /obj/item/kisser
+
+	if(HAS_TRAIT(user, TRAIT_KISS_OF_DEATH))
+		kiss_type = /obj/item/kisser/death
+
+	var/obj/item/kiss_blower = new kiss_type(user)
+	if(user.put_in_hands(kiss_blower))
+		to_chat(user, span_notice("You ready your kiss-blowing hand."))
+		user.balloon_alert_to_viewers("preparing kiss...", TRUE)
```
Comment (thgvr) [inline code/modules/mob/living/emote.dm]: this shows up to everyone, and shows "1" to yourself

**Comment by triplezeta** [comment]
this PR breaks giving

**Comment by Zevotech** [comment]
> this PR breaks giving

You are too far away! (x12)

**Comment by Sun-Soaked** [comment]
> this PR breaks giving

why am I cursed to deal with wsands code implementing giving in a completely different way to tg at the same time

anyway let me see what causes this

**Comment by Sun-Soaked** [comment]
I missed a single word. Guh

This should be fully functional now

**Comment by Sun-Soaked** [comment]
It wasn't whitesand's fault at all but I feel like it's responsible anyway

**Review by ZephyrTFA — APPROVED** [review]

**Pull request merged**